src/app/lib/config/services/amharic.js

```diff
@@ -64,7 +64,7 @@ const service = {
           message: 'የሚፈልጉትን ገፅ ማምጣት ባለመቻላችን እናዝናለን። እባክዎ እንደገና ይሞክሩ',
           solutions: ['በገፅዎ ላይ ያለውን ሪፍሬሽ የሚለውን ቁልፍ ይጫኑ', 'እንደገና ይመለሱ'],
           callToActionFirst: 'እንደ አማራጭም እባክዎ ገፃችን ላይ የሚገኘውን ',
-          callToActionLinkText: '[BBC ድረገፅ ይጎብ]',
+          callToActionLinkText: 'BBC ድረገፅ ይጎብ',
```

Comment:
Are you sure this is correct? Looking at the translations sheet it says `BBC News [service] ድረገፅ ይጎብኙ` so swapping out `[service]` would produce `BBC News አማርኛ ድረገፅ ይጎብኙ`

Reply:
Thanks for pointing

Reply:
I wonder if we need to make this consistent, or leave the translations that were already tested in the past. Some really do not follow the sheet literally; like Indonesian says 'Sebagai alternatif, coba klik Berita', but says 'Sebagai alternatif, coba klik [BBC News [service] homepage]' in the sheet. So the words BBC, homepage and the name of the service were dropped, and only the translation for the word News was made into the link.